sw/device/sca/ecc384_serial.c

```diff
@@ -0,0 +1,345 @@
+// Copyright lowRISC contributors.
```

Comment:
@bilgiday , for the beginning I would just merge one application. This will half the number of changes and reduce review effort. @vrozic mentioned that one doesn't seem to work on the FPGA currently. So maybe just select that one and leave the other for the moment. Once this one application is more stable and know exactly how things should be done it will be a piece of cake for you to add the other one.

Reply:
Thanks Pirmin. The problem was on the ot-sca side: I was using a single `*config.yaml` file for both of the applications. @vrozic and I changed the ot-sca code to use a separate yaml file for each application. Now, the host-side code (ot-sca) is running without any problems.

During the analysis phase, I will follow your advice and only focus on one application at a time.

Reply:
This sounds good to me.